Bindings/Python/CMakeLists.txt

```diff
@@ -6,6 +6,12 @@
 find_package(PythonInterp 2.7 REQUIRED)
 find_package(PythonLibs 2.7 REQUIRED)
 
+# We may need to update the python install dir to include the python version,
```

Comment:
The above `find_package` lines will find Python 3 if you have it on your computer. If one has both 2 and 3, I'm not sure which one is found. Also, the new install directory for the python packages will work for python 3.